pkgs/by-name/hy/hyprland-monitor-attached/package.nix

```
@@ -0,0 +1,24 @@
+{ rustPlatform, fetchFromGitHub, lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hyprland-monitor-attached";
+  version = "unstable-2023-10-03";
+
+  src = fetchFromGitHub {
+    owner = "coffebar";
+    repo = pname;
+    rev = "72551d65bfec5afa08f4053620415e6e5f64d0e1";
```

```suggestion
    rev = version;
```
Upstream tagged the latest version